docassemble/clcsa-01-service-directory/data/questions/main.yml

```
@@ -39,7 +116,7 @@ features:
   javascript:
     - custom-f2ddd3f2.js
   debug: False
-  hide standard menu: True
+  hide standard menu: False
```

By default DA uses a dark navbar, I think the person who modified the colour only changed bg colour and forgot to make sure text is actually visible (the colour is white by default). So we need to tell DA that we are using a light navbar
```suggestion
  hide standard menu: False
  inverse navbar: False
```